lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -245,6 +260,30 @@ class MetroItinerary<Props> extends NarrativeItinerary {
       currency
     )
 
+    const roundedCo2VsBaseline = Math.round(itinerary.co2VsBaseline * 100)
+    const emissionsNote = !mini &&
+      Math.abs(roundedCo2VsBaseline) >= (co2Config?.co2Cutoff || 0) &&
+      co2Config?.enabled && (
```

Comment:
Bit of a nitpick, but the `enabled` check can probably come first

Reply:
Additionally, do we need an explicit `enabled` flag? Can't we just use the presence of one or more `carbonIntensity` keys?

Reply:
Not really, because someone (probably most people) will just want to use the defaults. Technically we could check if the `co2` property exists in the config but I think that would look a little weird. You'd have the property like `co2:` in the yaml all on its own with nothing under it.

Reply:
That's a good point. I'll leave it up to the second reviewer to decide on this!